packages/bpk-component-calendar/src/BpkCalendarGridHeader.js

```diff
@@ -59,8 +59,8 @@ const WeekDay = props => {
   }
 
   return (
-    <Element className={classNames.join(' ')} title={weekDay.name}>
-      <span aria-hidden="true">{weekDay[weekDayKey]}</span>
+    <Element className={classNames.join(' ')} title={weekDay.name} aria-hidden>
+      <span>{weekDay[weekDayKey]}</span>
```

Comment:
Heading change - move `aria-hidden` up by one element. The `title` appears to be being read when using TalkBack otherwise.

Reply:
Would it make sense to just apply `aria-hidden="true"` to the entire header instead of each date?

Reply:
It possible - was aiming for as minimal chance of unspotted side effects as possible by moving just one level - but I have no objection if we wanted to move it higher.

Reply:
Moved up